app/containers/userSettings/index.tsx

```diff
@@ -12,6 +12,46 @@ const UserSettings: React.FC = () => {
     <>
       <div className={s.wrapper}>
         <h1 className={s.title}>Settings</h1>
+        <div
```

Comment:
1. The tabs have NO FEATURE yet. why they should be included in the pull request?
2. why inline-style?

Reply:
Oh my mistake.  I'll erase it.